test/unit/inet-address-tests.js

```diff
@@ -94,6 +94,19 @@ describe('InetAddress', function () {
       helper.assertInstanceOf(val, InetAddress);
       assert.strictEqual(val.toString(), '10.11.12.13');
     });
+    it('should parse IPv6 Addresses with Embedded IPv4 Addresses', function () {
+      [
+        ['0:0:0:0:0:FFFF:129.144.52.38',    '00000000000000000000ffff81903426'],
+        ['0:0:0:0:0:0:129.144.52.38',       '00000000000000000000000081903426'],
+        ['::ffff:129.144.52.38',            '00000000000000000000ffff81903426'],
+        ['::ffff:254.255.52.32',            '00000000000000000000fffffeff3420'],
+        ['::13.1.68.3',                     '0000000000000000000000000d014403'],
+        ['fe::13.1.68.3',                   '00fe000000000000000000000d014403']
```

Comment:
ok, sounds good, its a new implementation and there is no need to implement a deprecated spec.
[RFC 4291 (that obsoletes 3513) states](https://tools.ietf.org/html/rfc4291#section-2.5.5):

> The "IPv4-Compatible IPv6 address" is now deprecated because the
   current IPv6 transition mechanisms no longer use these addresses.

Defining IPv4-mapped as:
```
   |                80 bits               | 16 |      32 bits        |
   +--------------------------------------+--------------------------+
   |0000..............................0000|FFFF|    IPv4 address     |
   +--------------------------------------+----+---------------------+
```

We should explicitly validate anything different. Thanks for insisting on this!